pom.xml

```diff
@@ -123,6 +123,10 @@
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-stream</artifactId>
         </dependency>
+        <dependency>
```

Comment:
You can move in the compilation dep section (spring cloud is not at the right place too)

Reply:
done